oqs/README.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+Here the `openssl-sphincs.patch` file has our fork of the OQS (Open
+Quantum-Safe) project's fork of OpenSSL. The only files that have been changed
+in our fork are configuration and generated files.
+
+The reason we need this fork is that we have chosen to support a different set
+of SPHINCS+ variants than those that were chosen by the OQS team. As such, we
+enable and disable some variants and then re-generate code. We need the fork of
+OpenSSL because it generates our X.509 certificates for us and we do interop
+testing against it.
+
+This in turn requires that we use a specific version of liboqs as the OpenSSL
+fork expects SIKE and SIDH to be present, but it is no longer there.
+
+You can fetch OpenSSL source, apply our patch, build the code and then generate
+the certificates. Instructions for doing it all are below.
+
+# Building our Fork
+
+Below, you will find some simple instructions on how to build our fork of OQS's
+fork of OpenSSL.
+
+## liboqs
+
+NOTE: These are the same instructions that can be found in the wolfssl repo's
+      `INSTALL` file. If you already followed the instructions there, you can
+      skip this section.
+
+```
+$ mkdir ~/oqs
+$ cd ~/oqs
+$ git clone --single-branch https://github.com/open-quantum-safe/liboqs.git
+$ cd liboqs/
+$ git checkout af76ca3b1f2fbc1f4f0967595f3bb07692fb3d82
+$ mkdir build
+$ cd build
+$ make all
+$ sudo make install
+```
+
+At this point liboqs is properly installed.
+
+## OpenSSL
+
+```
+$ cd ~/oqs/
+$ git clone --single-branch --branch=OQS-OpenSSL_1_1_1-stable https://github.com/open-quantum-safe/openssl.git
+$ cd openssl
+$ git checkout e9160975eeb9796ff3550e8c2c35db63157a409b
+$ patch -p1 < /path/to/osp/oqs/openssl-sphincs.patch
+$ ./config no-shared
+$ make all
+```
+
+NOTE: There is no need to install this fork of OpenSSL.
+
+# Generating the Certificates
+
+We have scripts for generating certificate chains for all the variants of all
+the post-quantum algorithms that we support. simply copy them into the openssl
+directory and execute them to generate the certificate chains.
```

Comment:
Might be helpful to show what the output of running those scripts is. Also now you have duplicates of these here: https://github.com/wolfSSL/wolfssl-examples/tree/master/pq 
You should decide a single place to maintain these.